.eleventy.js

```diff
@@ -159,6 +159,79 @@ module.exports = function (config) {
     </svg>`;
   });
 
+  // Set the modal 
+  //
+  // Potential tech debt
+  //
+  // We're hardcoding all this stuff instead of using params to
+  // get this deployed quickly. If we use another modal on the site,
+  // we should consider refactoring this to a paired shortcode with
+  // multiple params. If we use 3+ different modals on the site, we
+  // HAVE to refactor this.
+  config.addLiquidShortcode('modal-start-application', function () {
+    return `<div class="margin-y-3">
+      <a
+        href="#modal-start-application"
+        class="usa-button"
+        aria-controls="modal-start-application"
+        data-open-modal
+      >Start a .gov domain request</a>
+      <div
+        class="usa-modal"
+        id="modal-start-application"
+        aria-labelledby="modal-start-application-heading"
+        aria-describedby="modal-start-application-description"
+      >
+        <div class="usa-modal__content">
+          <div class="usa-modal__main">
+            <h2 class="usa-modal__heading utility-header" id="modal-start-application-heading">
+            Requirements for requesting a .gov domain
+            </h2>
+            <div class="usa-prose">
+              <ul>
+                <li>You must be a government employee, or be working on behalf of a government organization, to request a .gov domain.</li>
+                <li><a href="https://login.gov/help/get-started/create-your-account/">You must have a Login.gov account</a>. Login.gov provides a simple and secure process for signing into many government services with one account.
+                  <ul>
+                    <li>Before you request your first .gov domain, you must verify your identity with Login.gov. This is a necessary layer of security that requires you to prove you are you, and not someone pretending to be you. <a href="https://beta.get.gov/help/account-management/">Get help with Login.gov</a>.</li>
+                  </ul>
+                </li>
+              </ul>
```

Comment:
@rachidatecs I think the beta.get.gov links need to be updated here so it doesn't conflict when merged with what @gabydisarli is doing

Reply:
Good call. Will change them to get.gov

Reply:
Done